app/src/main/java/ch/epfl/sdp/entity/EntityType.java

```diff
@@ -0,0 +1,17 @@
+package ch.epfl.sdp.entity;
+
+public enum EntityType {
+    ENEMY,
+    PLAYER,
+    ITEMBOX,
+    HEALTHPACK,
+    SCAN,
+    SHIELD,
+    SHRINKER,
+    USER,
+    MARKET,
+    TRAP,
+    SHELTER,
+    COIN,
+    NONE
+}
```

Comment:
This file should be deleted, we are not using EntityType anymore in the master branch

Reply:
Ah because we need it previously. But yeah I can delete it